github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -194,6 +194,49 @@ periodics:
     - name: gcs
       secret:
         secretName: gcs
+- name: periodic-kubevirt-e2e-k8s-1.19-sysprep
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+    testgrid-alert-email: bodnopoz@redhat.com, kubevirtci-sysprep-test@googlegroups.com
+    testgrid-num-failures-to-alert: "1"
+    testgrid-alert-stale-results-hours: "1"
+  cron: "0 0 * * *"
+  decorate: true
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: master
+    work_dir: true
+  cluster: phx-prow
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+        env:
+          - name: KUBEVIRT_QUARANTINE
+            value: "true"
+          - name: TARGET
+            value: "windows_sysprep"
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - "automation/test.sh"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "40Gi"
```

This is the current value we are using for e2e tests.
```suggestion
            memory: "29Gi"
```

---

We need more for this case considering the 32gb that the windows image takes. Is this a problem?

---

Nope, this is the RAM used by the job, not related to the image size.